release/_package.sh

```diff
@@ -120,23 +120,26 @@ function package_release {
 
 	lc_cd "liferay-${LIFERAY_RELEASE_PRODUCT_NAME}"
 
-	zip -qr "${_BUILD_DIR}/release/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-osgi-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.zip" osgi
-
 	lc_cd tomcat/webapps/ROOT
 
-	zip -qr "${_BUILD_DIR}/release/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.war" ./*
-
 	lc_cd "${_BUILD_DIR}/release/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}"
```

Comment:
az lc_cd sorok miért nem kerültek be az if-be?

Reply:
ezeket nem vettem észre így lehet az egész sorrend változik, javítom